components/forms/w-select.vue

```diff
@@ -1,26 +1,72 @@
 <template>
-  <w-field v-bind="{ ...$attrs, ...$props }" #default="{ triggerValidation }">
-    <div class="input mb-0">
-      <div class="input--select__wrap">
-        <select v-bind="{ ...$attrs, class: '' }" v-model="model" :id="id" @blur="triggerValidation">
+  <w-field v-bind="{ ...$attrs, ...$props }" #default="{ triggerValidation, hasValidationErrors }">
+    <div :class="wrapperClasses">
+      <div :class="selectWrapperClasses">
+        <select
+          :class="[
+            selectClasses,
+            {
+              [ccSelect.invalid]: hasValidationErrors,
+            }
+          ]"
+          v-bind="{ ...$attrs, class: '' }" v-model="model" :id="id" @blur="triggerValidation">
           <slot />
         </select>
+        <div :class="chevronClasses">
+          <svg
+              xmlns="http://www.w3.org/2000/svg"
+              width="16"
+              height="16"
+              fill="none"
+              viewBox="0 0 16 16"
+          >
+            <path
+              stroke="currentColor"
+              strokeLinecap="round"
+              strokeLinejoin="round"
+              strokeWidth="1.5"
+              d="M2.5 5.5L8 11L13.5 5.5"
+            />
+          </svg>
+        </div>
       </div>
     </div>
   </w-field>
 </template>
 
-<script>
+<script setup>
+import { computed, useAttrs } from 'vue';
+import { select as ccSelect } from '@warp-ds/component-classes';
 import { createModel } from 'create-v-model'
 import { default as wField, fieldProps } from './w-field.vue'
 
-export default {
-  name: 'wSelect',
-  components: { wField },
-  inheritAttrs: false,
-  props: fieldProps,
-  setup: (props, { emit }) => ({
-    model: createModel({ props, emit })
-  })
-}
+const props = defineProps(fieldProps);
+const emit = defineEmits(['update:modelValue']);
```

Comment:
I see it was part of w-textarea, but not w-select. I assume it was intended to add it here, but thought I'd doublecheck 😊

Reply:
Yea, it is the same for input, textarea and select. Needed to add it after I've refactored to use `script setup`

Reply:
Does it make sense?

Reply:
Yes! Thank you for the explanation!